### PR TITLE
FileResponse::prepareDownload fixed for 255 bytes filename limit

### PR DESCRIPTION
**Pull request by hranicka**
See http://en.wikipedia.org/wiki/Comparison_of_file_systems#Limits

I've changed only one method in whole Kdyby\Curl.
Maybe in some other places there is also "URL to file name converting".

But I've gained `Kdyby\Curl\FileNotWritableException` errors only for downloads and with this patch it's OK now.

Is this patch OK?

### CODE REVIEW

File: src/Kdyby/Curl/FileResponse.php [hunk 1]
```diff
@@ -214,4 +214,15 @@ public static function prepareDownload(CurlWrapper $curl, $dir)
 		));
 	}
 
+
+
+	/**
+	 * @param string $url
+	 * @return string
+	 */
+	public static function sanitizeFileName($url)
```
Comment (fprochazka) [inline src/Kdyby/Curl/FileResponse.php]: I don't this alias for sha1 has to be here. You could have just inlined it in the `prepareDownload` method.
Also, `md5()` should be faster.

Comment (hranicka) [inline src/Kdyby/Curl/FileResponse.php]: Ok, it's more simplier now :)

**Comment by fprochazka** [comment]
Thank you!

**Comment by hranicka** [comment]
Kdyby\Curl is really awesome PHP CURL wrapper ;)

**Pull request merged**